timecard.js

```diff
@@ -272,6 +273,58 @@ TimeCard.prototype.print = function(cb) {
 
 };
 
+/**
+ * Append .timecard.js to .gitignore file
+ *
+ */
+TimeCard.prototype.append = function(cb) {
+
+	var self = this;
+
+	// First check to see if the .gitignore file exists
+    fs.stat(process.cwd()+'/.gitignore', function(err, stats){
+
+    	if (err && err.code === 'ENOENT'){
+    		if (typeof cb === 'function') {
+    			cb(new Error(errors.noGitIgnoreFound));
+            	return;
+    		}
+        } else if (err) {
+            throw err;
+        }
+
+        if (stats && stats.isFile()){
+
+            var question = [{
+                    type: "confirm",
+                    name: "appendToGitIgnore",
+                    message: "A .gitignore file exists. Do you want to add .timecard.json to it?",
+                    default: true
+                }];
+
+            inquirer.prompt(question, function(answer) {
+                if (answer.appendToGitIgnore == true){
+
+                	var patterns = gitignore(process.cwd()+'/.gitignore');
+
+                	if (patterns.indexOf('.timecard.json') > -1){
+                		message.print(process.cwd()+'/.gitignore');
```

Comment:
we don't need the location of the .gitignore printed out, chances are the user is always going to be interfacing with timecard in the same directory as their git root.